web/template/course-overview.gohtml

```diff
@@ -45,6 +47,7 @@
             </div>
         </div>
         <!-- Next Lecture -->
+        {{if $course.HasNextLecture}}
```

Comment:
Since it's hidden (Great idea btw!) we could get rid of one if-branch in stream.FriendlyNextDate:
```
func (s Stream) FriendlyNextDate() string {
 // start remove
	if now.With(s.Start).Before(time.Now()) {
		return "No upcoming stream"
	}
// end
	if now.With(s.Start).EndOfDay() == now.EndOfDay() {
		return fmt.Sprintf("Today, %02d:%02d", s.Start.Hour(), s.Start.Minute())
	}
	if now.With(s.Start).EndOfDay() == now.With(time.Now().Add(time.Hour*24)).EndOfDay() {
		return fmt.Sprintf("Tomorrow, %02d:%02d", s.Start.Hour(), s.Start.Minute())
	}
	return s.Start.Format("Mon, January 02. 15:04")
}
```

Reply:
Good catch, I removed it. :+1: